lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -330,6 +334,13 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'Your New Relic [license key](/docs/apis/intro-apis/new-relic-api-keys/#ingest-license-key).'
         },
+        :api_key => {
+          :default => '',
+          :public => true,
+          :type => String,
+          :allowed_from_server => false,
+          :description => 'Your New Relic api key. Required to use New Relic REST API v2 for deployment task'
```

Comment:
The 2 instances of "API" should have matched capitalizations here. I think "API" in all-caps is best.

Reply:
addressed 5b7e76cc0f08701e2b5da74305327b312c74c59b